### PR TITLE
dumpifs: init at 0-unstable-2020-05-07

### PR DESCRIPTION
**Pull request by KSJ2000**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Noodlez1232** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 348768`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>dumpifs</li>
  </ul>
</details>

**Comment by Noodlez1232** [comment]
I couldn't test functionality, but it seems to build and run

**Review by FliegendeWurst — APPROVED** [review]

File: pkgs/by-name/du/dumpifs/package.nix
Comment (FliegendeWurst) [inline pkgs/by-name/du/dumpifs/package.nix:52]: You also need to add yourself to maintainer-list.nix :)

Comment (KSJ2000) [reply pkgs/by-name/du/dumpifs/package.nix:52]: I added myself in unyaffs pull request. After it is merged, the others should be good as well.

Comment (tomodachi94) [reply pkgs/by-name/du/dumpifs/package.nix:52]: Added in https://github.com/NixOS/nixpkgs/pull/348979, waiting on [a rebase](https://github.com/NixOS/nixpkgs/pull/348979#issuecomment-2475643855)

Comment (KSJ2000) [reply pkgs/by-name/du/dumpifs/package.nix:53]: Done for:
https://github.com/NixOS/nixpkgs/pull/348766
https://github.com/NixOS/nixpkgs/pull/349032
https://github.com/NixOS/nixpkgs/pull/349055
https://github.com/NixOS/nixpkgs/pull/348765
https://github.com/NixOS/nixpkgs/pull/348768

**Review by FliegendeWurst — APPROVED** [review]

**Pull request merged**
